client/src/app/components/comment-section/comment-section.component.ts

```diff
@@ -0,0 +1,126 @@
+import { Component } from '@angular/core';
+import { CommentDto } from '../../../shared/models/CommentResponseDto';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+import { CommentsService } from '../../../services/comment-service/comment.service';
+import { ActivatedRoute } from '@angular/router';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ReactiveFormsModule } from '@angular/forms';
+import { CommentsHubService } from '../../../services/comment-hub-service/comment-hub.service';
+import { createCommentDto } from '../../../mappers/createCommentDto';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './comment-section.component.html',
+  styleUrls: ['./comment-section.component.css'],
+  standalone: true,
+  imports: [CommonModule, FormsModule, ReactiveFormsModule],
+})
+export class CommentSectionComponent {
+  title = 'angular-client';
+  content: string = '';
+  comments: CommentDto[] = [];
+  editingCommentIndex: number | null = null;
+  taskId: string = '';
+  commentForm: FormGroup;
+
+  constructor(
+    private commentService: CommentsService,
+    private route: ActivatedRoute,
+    private fb: FormBuilder,
+    private commentHubService: CommentsHubService
+  ) {
+    this.commentForm = this.fb.group({
```

Comment:
usually forms are used in such places where many inputs, textareas and so on. to store one value you dont need to define form